modules/eks/submodules/k8s/templates/k8s-functions.sh.tftpl

```diff
@@ -119,8 +119,9 @@ install_calico() {
       --namespace "tigera-operator" \
       --set installation.kubernetesProvider=EKS \
       --set installation.cni.type=AmazonVPC \
-      --set installation.registry="quay.io/" \
+      --set installation.registry="${calico_image_registry}/" \
       --set installation.fipsMode="${calico_fips_mode}" \
+      --wait \
```

Comment:
I also don't totally remember if not setting this was on purpose. I can imagine it is if say calico-node takes a while to rolling update